op-node/l2/source.go

```diff
@@ -218,16 +219,16 @@ func blockToBlockRef(block *types.Block, genesis *rollup.Genesis) (eth.L2BlockRe
 }
 
 type ReadOnlySource struct {
-	rpc     *rpc.Client       // raw RPC client. Used for methods that do not already have bindings
-	client  *ethclient.Client // go-ethereum's wrapper around the rpc client for the eth namespace
+	rpc     client.RPC    // raw RPC client. Used for methods that do not already have bindings
+	client  client.Client // go-ethereum's wrapper around the rpc client for the eth namespace
 	genesis *rollup.Genesis
 	log     log.Logger
```

Comment:
Planning to potentially unify this with the other L2 source bindings, or at least reduce duplication. Thank you for integrating metrics though.